builder/js/ncf.js

```diff
@@ -840,8 +944,8 @@ $scope.groupMethodsByCategory = function () {
       // Transform back ncfTechnique to UITechnique, that will make it ok
       var savedTechnique = toTechUI(ncfTechnique);
 
-       // Update technique if still selected
-      if (angular.equals($scope.originalTechnique, origin_technique)) {
+      // Update technique if still selected
+      if (angular.equals($scope.originalTechnique, origin_technique) && !$scope.updatedTechnique) {
```

Comment:
updated technique may not be a boolean, never check like this, do check on undefined